src/revlanguage/workspace/RbRegister_Dist.cpp

```diff
@@ -409,6 +410,9 @@ void RevLanguage::Workspace::initializeDistGlobalWorkspace(void)
         // constrained topology distribution
         AddDistribution< TimeTree                   >( new Dist_ConstrainedTopology() );
 
+        // constrained topology distribution
```

Comment:
Need to write help for new dnConstrainedTopologySSE

Reply:
There's now a blank help file, so at least we've got that! May need to actually write the help for both SEE and regular distributions.